content/com/github/ldapchai/ldapchai-0.8.6.jar.diffoscope

```diff
@@ -0,0 +1,1068 @@
+│ │ +  #370 = Methodref          #28.#371      // com/novell/ldapchai/provider/WireTraceWrapper.lambda$objectToDebugString$4:(Ljava/lang/Exception;)Ljava/lang/String;
+│ │ +  #371 = NameAndType        #336:#337     // lambda$objectToDebugString$4:(Ljava/lang/Exception;)Ljava/lang/String;
+│ │ +  #372 = Utf8               InnerClasses
+│ │ +  #373 = Class              #374          // com/novell/ldapchai/provider/ChaiProvider
+│ │ +  #374 = Utf8               com/novell/ldapchai/provider/ChaiProvider
+│ │ +  #375 = Utf8               LdapOperation
+│ │ +  #376 = Class              #377          // java/lang/invoke/MethodHandles$Lookup
+│ │ +  #377 = Utf8               java/lang/invoke/MethodHandles$Lookup
+│ │ +  #378 = Class              #379          // java/lang/invoke/MethodHandles
+│ │ +  #379 = Utf8               java/lang/invoke/MethodHandles
+│ │ +  #380 = Utf8               Lookup
+│ │  {
+│ │    private static final com.novell.ldapchai.util.internal.ChaiLogger LOGGER;
+│ │      descriptor: Lcom/novell/ldapchai/util/internal/ChaiLogger;
+│ │      flags: (0x001a) ACC_PRIVATE, ACC_STATIC, ACC_FINAL
+│ │  
+│ │    private final java.util.concurrent.atomic.AtomicLong operationCounter;
+│ │      descriptor: Ljava/util/concurrent/atomic/AtomicLong;
+│ │ @@ -420,61 +417,61 @@
+│ │          31: invokevirtual #37                 // Method com/novell/ldapchai/util/internal/ChaiLogger.warn:(Ljava/util/function/Supplier;)V
+│ │          34: new           #43                 // class java/lang/IllegalStateException
+│ │          37: dup
+│ │          38: ldc           #25                 // String attempt to obtain WireTrace wrapper when watchdog is not enabled in chai config
+│ │          40: invokespecial #45                 // Method java/lang/IllegalStateException."<init>":(Ljava/lang/String;)V
+│ │          43: athrow
+│ │          44: aload_0
+│ │ -        45: invokeinterface #49,  1           // InterfaceMethod com/novell/ldapchai/provider/ChaiProviderImplementor.getClass:()Ljava/lang/Class;
+│ │ -        50: invokestatic  #53                 // Method java/lang/reflect/Proxy.isProxyClass:(Ljava/lang/Class;)Z
+│ │ -        53: ifeq          76
+│ │ -        56: aload_0
+│ │ -        57: instanceof    #28                 // class com/novell/ldapchai/provider/WireTraceWrapper
+│ │ -        60: ifeq          76
+│ │ -        63: getstatic     #27                 // Field LOGGER:Lcom/novell/ldapchai/util/internal/ChaiLogger;
+│ │ -        66: invokedynamic #59,  0             // InvokeDynamic #1:get:()Ljava/util/function/Supplier;
+│ │ -        71: invokevirtual #37                 // Method com/novell/ldapchai/util/internal/ChaiLogger.warn:(Ljava/util/function/Supplier;)V
+│ │ +        45: invokevirtual #49                 // Method java/lang/Object.getClass:()Ljava/lang/Class;
```

Comment:
`invokeinterface` is replaced by `invokevirtual`. I thought this was a build environment problem, but when I built with `jdk=8`, I got an error but I did not investigate.